frontend/src/components/Layout.js

```diff
@@ -10,38 +9,16 @@ const Wrapper = styled.div`
 const Content = styled.main`
   height: 100vh;
   margin: 0 auto;
-  padding: 5em 2em;
+  padding: 7em 5em;
```

Comment:
when I saw the preview, it seemed like the Content was not left aligned with the "vocab buddy" which is what I think was in the Figma

Reply:
Ohh oh maybe ill change that heheh thanks for catching it